docs/workflows.md

```diff
@@ -78,7 +84,8 @@ Set `Workflow permissions` to `Read and write permissions`, then hit `Save`:
 
 ### Set up itch.io upload
 
-Create your itch.io page with `Kind of project = HTML`, and double-check that the [`ITCH_TARGET` variable](#set-up-variables) you've set up earlier matches your itch.io page.
+Create your itch.io page with the same name as is used in the `ITCH_TARGET` variable in [release.yaml](../.github/workflows/release.yaml).
```

Comment:
Should still specify `Kind of project = HTML`

Reply:
Whoops

Reply:
[`6021871` (#163)](https://github.com/TheBevyFlock/bevy_quickstart/pull/163/commits/60218717460e7337d668dd0a1d78b6b067fc4159)